docs/source/daml/reference/packages.rst

```diff
@@ -77,12 +77,10 @@ older SDK, by listing them under the ``data-dependencies`` stanza:
   data-dependencies:
     - /home/jondoe/bar-0.0.0/.daml/dist/bar-0.0.0.dar
 
-Modules from data dependencies can be imported as usual via a ``import Foo`` statement. To have the
-``Template``, ``Choice``, ``TemplateKey``, ``Eq``, ``Show`` and ``HasField`` available of the imported template
-types, also import the generated instances module:
+Modules from data dependencies can be imported as usual, but need to be qualified by the
```

Comment:
I am not sure I understand why this is required. Which module do we collide with? Afaict we should only be loading interface files for the `instances` package but not from the original package (which might not even have interface files). I would prefer if we could avoid having the instances module leak to users.

Reply:
discussed this with @hurryabit . the imports are ambiguous without package qualification because the original module is also in the package database. we might get away by removing it from the package database after having built the interface package. but we'd like to go with this for now and see if we can drop the qualification in a later step.